package.json

```diff
@@ -13,7 +13,9 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "find-config": "^1.0.0",
+    "fs": "0.0.1-security",
```

Comment:
You shouldn't need to install fs, it's a default module?

Reply:
yeah it is, i'll take it out